### PR TITLE
Operations layer implementation.

### PR DESCRIPTION
**Pull request by sjames-iberis**
Implements rpc calls defined in RFC6241 - Netconf Configuration Protocol.  

Also, supports GetSchemas and GetSchema from RFC6022 - Yang Module for NETCONF Monitoring.

### CODE REVIEW

File: v2/netconf/common/model.go [hunk 1]
```diff
@@ -63,10 +63,25 @@ type NotificationMessage struct {
 	Event     Notification `xml:",any"`
 }
 
+type Union struct {
+	ValueStr interface{}
+	ValueXml string `xml:",innerxml"`
```
Comment (damianoneill) [inline v2/netconf/common/model.go]: should this not be a pointer?

File: v2/netconf/ops/model.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
+package ops
+
+import "encoding/xml"
+
+const (
+	// Configuration Datastores
```
Comment (damianoneill) [inline v2/netconf/ops/model.go]: there are additional datastores.  https://tools.ietf.org/html/rfc8342

File: v2/netconf/ops/session.go [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+type Namespace struct {
+	Id   string
+	Path string
+}
+
+// OpSession represents a Netconf Operations OpSession
+type OpSession interface {
+	client.Session
+
+	// GetReq issues a GET request, with the supplied subtree filter and stores the response in the result, which
```
Comment (damianoneill) [inline v2/netconf/ops/session.go]: Comment name wrong

File: v2/netconf/common/model.go (hunk 1 above)
Comment (sjames-iberis) [reply v2/netconf/common/model.go]: No change required.

File: v2/netconf/ops/model.go (hunk 2 above)
Comment (sjames-iberis) [reply v2/netconf/ops/model.go]: Added intended and operational, per rfc.

**Pull request merged**